templates/.semaphore/semaphore.yml

```diff
@@ -40,16 +35,14 @@ blocks:
       prologue:
         commands:
           - checkout
-          - cache restore bundle-$SEMAPHORE_GIT_BRANCH-$(checksum Gemfile.lock),bundle-$SEMAPHORE_GIT_BRANCH,bundle-master
-          - cache restore bundleconfig-$SEMAPHORE_GIT_BRANCH-$(checksum Gemfile.lock)
-          - cache restore yarn-$SEMAPHORE_GIT_BRANCH-$(checksum yarn.lock),yarn-$SEMAPHORE_GIT_BRANCH,yarn-master
-          - cache restore nodemodules-$SEMAPHORE_GIT_BRANCH-$(checksum package-lock.json),node-modules-$SEMAPHORE_GIT_BRANCH,node-modules-master
+          - cache restore
+          - bundle install --deployment --path vendor/bundle
```

Comment:
[This seems to be necessary:](https://docs.semaphoreci.com/article/99-rails-continuous-integration)
![image](https://user-images.githubusercontent.com/42026584/67667630-81264b00-f96e-11e9-9d39-8e57a1b39cf3.png)